src/Magnum/LibOVRIntegration/Conversion.h

```diff
@@ -0,0 +1,157 @@
+#ifndef Magnum_LibOvrIntegration_Conversion_h
+#define Magnum_LibOvrIntegration_Conversion_h
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2015
+              Jonathan Hale <squareys@googlemail.com>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/** @file
+ * @brief Converters and conversion methods.
+ *
+ * This header containts tools to convert libOVR types to magnum types.
+ *
+ * @author Jonathan Hale (Squareys)
+ */
+
+#include <OVR/OVR_CAPI.h>
+
+#include <Magnum/Magnum.h>
+#include <Magnum/Texture.h>
+#include <Magnum/Math/Range.h>
+#include <Magnum/Math/Vector.h>
+#include <Magnum/Math/Matrix4.h>
+#include <Magnum/Math/Quaternion.h>
+#include <Magnum/Math/DualQuaternion.h>
+
+namespace Magnum { namespace Math { namespace Implementation {
+
+// ovrSizei
```

Comment:
Please use `/* .. */` comments rather than `// ...` (except in Doxygen blocks, because these comments cannot be nested). 

(Explanation: I have such policy that I'm using `// ...` *exclusively* for quick-and-dirty coding (commenting-out some part of code, short-lived todo/fixme comments etc.) so when I see it in a commit, it subconsciously triggers a warning that there is something that shouldn't be here.)